source/includes/_tipser-elements-analytics.md

```diff
@@ -38,6 +38,21 @@ document.addEventListener('tw-track', function (e) {
 });
 ```
 
+If you want to track `tw-track` events in your Google Analytics, you can use this code snippet:
+
+```javascript
+document.addEventListener('tw-track', function(e) {
+    ga('send', {
+        hitType: 'event',
+        eventCategory: e.detail.description,
+        eventAction: e.detail.action,
+        eventLabel: e.detail.target,
```

Comment:
is there a good way of sending `e.detail.object` to Google Analytics as well ? Or is it not necessary for some reason ?

Reply:
@chmurson good point, however we were not sure with Mateusz, to which GA fields to pass the specific fields from `e.detail.object`. Usually GA custom dimensions are used for that and we can't make any assumptions about those on the POS side.